packages/nc-gui/composables/useApi/index.ts

```diff
@@ -3,14 +3,16 @@ import { Api } from 'nocodb-sdk'
 import type { Ref } from 'vue'
 import type { CreateApiOptions, UseApiProps, UseApiReturn } from './types'
 import { addAxiosInterceptors } from './interceptors'
-import { BASE_URL, createEventHook, extractSdkResponseErrorMsg, ref, unref, useCounter, useGlobal, useNuxtApp } from '#imports'
+import { BASE_FALLBACK_URL, createEventHook, extractSdkResponseErrorMsg, ref, unref, useCounter, useNuxtApp } from '#imports'
 
-export function createApiInstance<SecurityDataType = any>({ baseURL = BASE_URL }: CreateApiOptions = {}): Api<SecurityDataType> {
-  const { appInfo } = $(useGlobal())
+export function createApiInstance<SecurityDataType = any>({
+  baseURL = BASE_FALLBACK_URL,
+}: CreateApiOptions = {}): Api<SecurityDataType> {
+  const config = useRuntimeConfig()
 
   return addAxiosInterceptors(
     new Api<SecurityDataType>({
-      baseURL: baseURL ?? appInfo.ncSiteUrl,
+      baseURL: config.public.ncBackendUrl || baseURL,
```

Comment:
Verify this change with `nuxt generate` once

Reply:
Docker build seems to be working fine

<img width="1920" alt="image" src="https://user-images.githubusercontent.com/35809690/201016008-cf4d5c6f-8b3e-4e0f-b2a0-7438a932de04.png">
<img width="460" alt="image" src="https://user-images.githubusercontent.com/35809690/201016324-389a2ed1-dc2c-4edd-8cb7-ad9c5a13749d.png">